### DISCUSSION TITLE
Ipsec leftsubnet problems

### DISCUSSION THREAD

**Discussion by killgapwalker** [strongswan/strongswan#478] [General]
I can't encapsulate leftsubnet and left default route those both connections Going in 0.0.0.0 and doesnt INPUTing or prerouting other variations with private IP notify TS_unacceptable no Child Sa

**Comment by killgapwalker** [comment]
> I can't encapsulate leftsubnet and left default route those both connections Going in 0.0.0.0 and doesnt INPUTing or prerouting. Other variations with left route as private IP notify TS_unacceptable no Child Sa.

**Reply by tobiasbrunner** [reply]
Sorry, no idea what you are trying to say. But please follow #196 when posting here.
